tests/ansible_module_playook_test.go

```diff
@@ -0,0 +1,46 @@
+package tests_test
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+)
+
+const (
+	ansibleModuleURL          = "https://github.com/kubevirt/ansible-kubevirt-modules/"
+	ansibleModulePath         = "ansible-kubevirt-modules/lib/ansible/modules/clustering/kubevirt/"
+	ansibleModuleClonePath    = "ansible-kubevirt-modules"
+	ansibleModulePlaybookPath = "ansible-kubevirt-modules/tests/playbooks/"
+	ansibleModuleTestCfg      = "ansible-kubevirt-modules/tests/ansible.cfg"
+	ansibleModuleLogPath      = ansibleModuleClonePath + "ansible.log"
+)
+
+var _ = FDescribe("Run test playbooks from KubeVirt-Ansible module", func() {
+
+	Context("Test playbook ", func() {
+		DescribeTable("Playbook name:", func(playbookName string) {
+			// use Config file of the module
+			os.Setenv("ANSIBLE_CONFIG", ansibleModuleTestCfg)
+
+			cmd := exec.Command("ansible-playbook", "-vvvv", ansibleModulePlaybookPath+playbookName)
+			ok := cmd.Run()
+			if ok != nil {
+				fmt.Print(ok)
+				Fail("Playbook command returned with error")
```

Comment:
This is confusing first we print `ok` and later we say that command returned with error. I think we could reconsider `ok` name.

Reply:
updated.